packages/openfeature-server-provider/src/ConfidenceServerProvider.e2e.test.ts

```diff
@@ -1,36 +1,18 @@
-import { OpenFeature, ProviderEvents } from '@openfeature/server-sdk';
-import axios from 'axios';
+import { OpenFeature } from '@openfeature/server-sdk';
 import { createConfidenceServerProvider } from './factory';
 
 describe('ConfidenceServerProvider E2E tests', () => {
-  beforeAll(() => {
+  beforeEach(() => {
     const confidenceProvider = createConfidenceServerProvider({
-      fetchImplementation: async (url, request): Promise<Response> => {
-        return await axios.post(url as string, request?.body).then(
-          resp =>
-            ({
-              json: async () => {
-                return resp.data;
-              },
-            } as Response),
-        );
-      },
+      fetchImplementation: fetch,
```

Comment:
what's changing that previously we were waiting for the ready signal and now we are not?

Reply:
I don't know why we were waiting for the ready signal before. This is dynamic context (server) so it doesn't really make sense to wait for ready.